lib/text_parser.dart

```diff
@@ -82,17 +82,26 @@ const SUPPORTED_BLOCK_ELEMENTS = <String>{
 };
 
 class TextParser extends StatelessWidget {
-  TextParser({
+  const TextParser({
+    Key? key,
     this.shrinkToFit = false,
-    this.onLinkTap,
+    @Deprecated('Use [beforeLaunch] instead.') this.onLinkTap,
+    this.beforeLaunch,
     this.renderNewlines = false,
     required this.html,
     this.onImageError,
+    // keep in sync with [Html.linkHoverStyle]
     this.linkStyle = const TextStyle(
       decoration: TextDecoration.underline,
       color: Colors.blueAccent,
       decorationColor: Colors.blueAccent,
     ),
+    // keep in sync with [Html.linkHoverStyle]
```

Comment:
uuuuh......do you mean keep in sync with linkStyle and for linkStyle to keep in sync with the normal style?

Reply:
Well, in case we should change the default linkStyle, we should correspondingly adjust the linkHoverStyle too. That's all...

Reply:
well, the comments currently say to keep Html.linkHoverStyle in sync with itself ^^"

Reply:
aaaaah